### PR TITLE
Add support for python files with descriptions

### PR DESCRIPTION
**Pull request by Tyilo**
Supports the syntax:

```
'''
 * Author: Asger Hautop Drewsen
 * License: CC0
 * Description: Prints Foo
 * Status: Works
 * Time: $O(1)$
'''

print('Foo')
```

### CODE REVIEW

File: build/kactlprocessor.py [hunk 1]
```diff
@@ -84,18 +99,18 @@ def processwithcomments(caption, instream, outstream, listingslang = None):
...
     end = 0
     commands = {}
     while start >= 0 and not error:
         nsource = nsource.rstrip() + source[end:start]
-        end = source.find("*/", start)
+        end, endlen = find_end_comment(source, start + 1)
         if end<start:
             error = "Invalid /** */ comments."
             break
         comment = source[start+3:end].strip()
```
Comment (simonlindholm) [inline build/kactlprocessor.py]: so that this `start+3` becomes ... `start_end`? `start2`? something like that

File: build/kactlprocessor.py [hunk 2]
```diff
@@ -52,6 +52,21 @@ def addref(caption, outstream):
...
+        if i != -1 and (i < first[0] or first[0] == -1):
+            first = (i, len(s))
+
+    return first
+
+def find_start_comment(source, start=None):
+    return find_helper(source, start, ['/**', "'''"])
+
+def find_end_comment(source, start=None):
+    return find_helper(source, start, ['*/', "'''"])
```
Comment (simonlindholm) [inline build/kactlprocessor.py]: is it worth supporting `"""`? should we try to pair comment end syntax with the matching start syntax?

**Comment by Tyilo** [comment]
@simonlindholm Should be better now.
It would be better if it was language-specific, but I think it's good enough for now.

**Comment by simonlindholm** [comment]
Very nice!

**Pull request merged**